cmd/crates/soroban-spec-typescript/src/project_template/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "buffer": "6.0.3",
-    "@stellar/stellar-sdk": "12.1.0"
+    "@stellar/stellar-sdk": "12.2.0"
```

Comment:
I just dropped [`v12.3.0`](https://github.com/stellar/js-stellar-sdk/releases/tag/v12.3.0), sorry :laughing:

Reply:
Oh great! Updated for 12.3.0